tests/project.bats

```diff
@@ -27,4 +27,8 @@ setup() {
 	assert_output --partial "generated by dagger"
 
 	test ! -f ./cue.mod/pkg/.gitignore
+
+
+	"$DAGGER" project info
+	assert_output --partial "Current dagger project in"
```

Comment:
Perhaps you could also check that `"$TEMPDIR"` is part of the output?

Reply:
Added! Also addressed linting issues. If there is anything else I should change please let me know.